addons/pos_sale/static/tests/tours/pos_sale_tours.js

```diff
@@ -23,4 +23,20 @@ odoo.define('pos_sale.tour', function (require) {
     Chrome.do.clickTicketButton();
 
     Tour.register('PosSettleOrder', { test: true, url: '/pos/ui' }, getSteps());
+
+    startSteps();
+
+    ProductScreen.do.confirmOpeningPopup();
+    ProductScreen.do.clickQuotationButton();
+    ProductScreen.do.selectFirstOrder();
+    ProductScreen.do.clickOrderline("Product B", 1);
+    ProductScreen.do.pressNumpad('Qty 0');
+    ProductScreen.do.clickOrderline("Product A", 1);
+    ProductScreen.do.pressNumpad('Qty 1');
+    ProductScreen.do.clickPayButton();
+    PaymentScreen.do.clickPaymentMethod('Bank');
+    PaymentScreen.do.clickValidate();
```

Comment:
Before clicking the validate button, insert a step that checks the remaining amount to pay.

```
    PaymentScreen.check.remainingIs('0.0');
```